src/pages/Embeds/Plio.vue

```diff
@@ -304,6 +304,12 @@ export default {
   },
   computed: {
     ...mapGetters("auth", ["isAuthenticated"]),
+    /**
+     * Whether the scorecard will be shown or not
```

Comment:
shown - both yes or no - is possible even when enabled.
```suggestion
     * whether the scorecard is enabled or not
```